PlayCover/Utils/Cacher.swift

```diff
@@ -8,16 +8,42 @@
 import Foundation
 import AppKit
 import DataCache
+import CachedAsyncImage
 
 class Cacher {
-
+    static let shared = Cacher()
```

Comment:
Since a shared instance of Cacher has been created you need to reflect this change into the codebase as in:
PlayCover/Views/StoreInfoAppView.swift
PlayCover/Views/App Views/StoreAppView.swift
PlayCover/Views/App Views/PlayAppView.swift